.circleci/config.yml

```diff
@@ -0,0 +1,240 @@
+version: 2.1
```

Comment:
Is this `config.yml` from dwave-hybrid? I'd like to update build/test jobs in orb-ocean accordingly, and simplify CI configs in ocean packages. What do you think?

Reply:
Yup, and works for me